mimic/rest/maas_api.py

```diff
@@ -30,6 +30,16 @@
 Request.defaultContentType = 'application/json'
 
 
+def _find_index_of_id(xs, val):
```

Comment:
Anyway this is *totally* a quibble about an internal implementation detail now so I think I'll merge, you can refactor this later if you like my suggestion.